packages/wallets/src/components/DesktopWalletsList/DesktopWalletsList.tsx

```diff
@@ -17,12 +17,14 @@ const DesktopWalletsList: React.FC = () => {
         <div className='wallets-desktop-wallets-list'>
             {isLoading && <WalletsCardLoader />}
             {!isLoading && (
-                <WalletsContainer
-                    key={activeWallet && `wallets-card-${activeWallet?.loginid}`}
-                    renderHeader={() => <WalletListCard />}
-                >
-                    <AccountsList />
-                </WalletsContainer>
+                <React.Fragment>
+                    <WalletsContainer
+                        key={activeWallet && `wallets-card-${activeWallet?.loginid}`}
+                        renderHeader={() => <WalletListCard />}
+                    >
+                        <AccountsList />
+                    </WalletsContainer>
+                </React.Fragment>
```

Comment:
You can remove the fragment because its only one component

Reply:
Updated